LoyaltyProgram_Intern/src/main/java/com/pgs/soft/PasswordValidator.java

```diff
@@ -0,0 +1,67 @@
+package com.pgs.soft;
+
+import javax.servlet.http.HttpSession;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import com.pgs.soft.domain.User;
+import com.pgs.soft.dto.PasswordDTO;
+import com.pgs.soft.service.UserService;
+
+@Component
+public class PasswordValidator implements Validator {
+
+	@Autowired
+	private HttpSession session;
+	
+	@Autowired 
+	private UserService userService;
+	
+	@Override
+	public boolean supports(Class<?> clazz) {
+		
+		return clazz.equals(PasswordDTO.class);
+	}
+
+	//Metoda ustawiająca parametry validatora.
+	@Override
+	public void validate(Object target, Errors errors) {
+		PasswordDTO passwordDTO = (PasswordDTO) target;
+		validatePassword(errors, passwordDTO);
+		
+	}
+	
+	//Warunki validacji.
+	private void validatePassword(Errors errors, PasswordDTO passwordDTO) {
+		
+		if( !(passwordDTO.getOldPassword() != null && !passwordDTO.getOldPassword().isEmpty()) )
```

Comment:
Sure, I read about StringUtils, but it seemed to me a little too strong for just this operation.